scrapers/RawAttack.yml

```diff
@@ -0,0 +1,19 @@
+name: RawAttack
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - rawattack.com/updates/
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //h1[@class="title-scene"]/text()
+      Details: //p[@class="description-scene"]/text()
+      Performers:
+        Name: //p[@class="featuring"]/a
```

Comment:
This returns also the dot `.` after the performers name.
Switching to 
```yaml
      Performers:
        Name:
          selector: //p[@class="featuring"]/a
          postProcess:
            - replace:
                - regex: \.$
                  with: ""
```
cleans that.